### ISSUE TITLE
float value

### ISSUE THREAD

**Issue by RBEGamer** [homebridge/HAP-NodeJS#78]
is it possible to a a float characteristic with a specific name ?

**Comment by snowdd1** [comment]
You want to "add" a specific variable to use later in characteristics.js or in a callback?

**Comment by RBEGamer** [comment]
yes i want to add a float value to a accessory with a name e.g. "effect amount", if i add a device with 3 brightness sliders each slider has the name "Brightness"..

**Comment by snowdd1** [comment]
Ok. You want to have a custom name for the characteristic. As far as I know that is not supported yet, however I read rumors about more custom types are introduced in iOS 9. 
Someone here to leak some more ?

**Comment by RBEGamer** [comment]
in the Apple HomeKit Simulator you can add a custom type with different formats like bool int float..

**Comment by snowdd1** [comment]
I think the issue is we don't have the string constants for such types. Have a look at accessories/types.js, that is the list we're working with at the moment. If you can deduct values from the simulator, please post them here!

**Comment by RBEGamer** [comment]
![2](https://cloud.githubusercontent.com/assets/9280991/8366552/1db01f84-1b96-11e5-8fbf-192af18a12b8.png)
![1](https://cloud.githubusercontent.com/assets/9280991/8366555/22ba351e-1b96-11e5-994d-798a6e7bd0a7.png)

**Comment by snowdd1** [comment]
Ok, I tried with the UUID-part of "EC" as custom type.
It does work (with the light_accessory.js example, however, I did not find a way to give this custom property a name that would show up in the ios app. With myTouchHome, it shows just a slider with no label.
My repo is currently messed up (I need to separate my KNX development from HAP development so I cannot prepare a push right now. Changes required are a new entry in accessories/types.js:
<code>
exports.CUSTOM_CHARACTERISTIC_CTYPE = stPre + "EC" + stPost;
</code>
